buddyboss/buddyboss-platform
# The subscriber does not appear as a group leader from the Learndash dashboard, after creating group from the BB front-end.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2223 

### How to test the changes in this Pull Request:

1. Login as Subscriber.
2. Create group
3. Now login as admin
4. Go to LD group
5. Go to new created group.
6. Check user roles.

https://www.loom.com/share/aee656c0515e40c6b8a56acac040ee53

### Proof Screenshots or Video

https://www.loom.com/share/352335e3a5274d58a0a175b321979064

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
